modules/tensor_mechanics/src/materials/RadialReturnStressUpdate.C

```diff
@@ -106,6 +110,39 @@ RadialReturnStressUpdate::updateState(RankTwoTensor & strain_increment,
   stress_new = elasticity_tensor * (strain_increment + elastic_strain_old);
 
   computeStressFinalize(inelastic_strain_increment);
+
+  if (compute_full_tangent_operator &&
+      getTangentCalculationMethod() == TangentCalculationMethod::PARTIAL)
+  {
+    if (MooseUtils::absoluteFuzzyEqual(scalar_effective_inelastic_strain, 0.0))
+      tangent_operator.zero();
```

Comment:
This is the case already. `ComputeMultipleInelasticStress` keeps a vector of bools that only sets `compute_full_tangent_operator` to true if it is on the jacobian, if the model isn't only elastic, and the user set `tangent_operator=nonlinear`.

Reply:
Yeah, but it could be further optimized to treat the inelastic model as elastic if there's no inelastic strain. Premature optimization is the root of all evil, though, so let's just drop this for now.

Reply:
Oh I see. There's no cross communication between multiple creep models though. It'd have to make sure all creep models have zero inelastic strain...which I imagine is rare, if it happens at all beyond time = 0.